types.ts

```diff
@@ -1342,6 +1346,25 @@ export interface AssistantMessageTool extends BaseTool<ToolType.AssistantMessage
  */
 export interface SummarizerTool extends BaseTool<ToolType.Summarizer> {}
 
+/**
+ * @hidden
+ */
+export interface MCPTool extends BaseTool<ToolType.MCP> {}
```

Comment:
Should this tool type have an option to limit the scope to a set of named servers? Not a launch requirement, but would match what we do with `Pack` tools the `formulas` field.

Reply:
Yeah sure I can add that. Right now the upload validation limits to one mcp server per pack though.